add-link

mannypanwar opened the pull request:
#35 
added link to hacktoberfest in container logo and footer logo

mannypanwar committed:
add-link

added link to hacktoberfest in container logo and footer logo

anktrj reviewed (changes requested):
No need to change the link for the label "IIITR Hacktober Fest". Just Change the link for HactoberFest official icon.
Re config these changes and you are good to go. Feel free to ask for any assistance.
Thanks.

anktrj commented:
you can also checkout https://iiitranchi.github.io/hacktoberfest-19/attendees/ . Changes are requested here to.

mannypanwar committed:
add-link

Added link to hacktoberfest logo and removed link from label "IIITR Hacktober fest"

mannypanwar commented:
Made a new commit with hacktoberfest link in its official logo and removed the link from label "IIITR Hacktober Fest"
